### ISSUE TITLE
angularjs resource does not in Lightning Out context

### ISSUE THREAD

**Issue by AndreyVol** [forcedotcom/aura-ng#4]
Disclaimer: this may not be an issue but my lack of knowledge.

I am testing how aura-ng library works with Lightning Out. I am using Skip Saul's example: https://github.com/skipsauls/heroku-lightning (created my own copy of it).

When I try to load the demoComponent it does not load /resource/angularjs. I get the following error: 

Failed to load resource: the server responded with a status of 404 (Not Found): https://avlightingoutdemo.herokuapp.com/resource/auranglogo Failed to load resource: the server responded with a status of 404 (Not Found)

I think this is happening because of relative path in the regionController (line 16). I hacked it with a terrible hack hardcoding the resource Url and it worked. 

    init: function(component, event, helper) {
        var scripts = component.get("v.scripts");
        //scripts.unshift("/resource/angularjs");
        //this is an ugly hack to overcome a problem in lighting out scenario where this relative resource path does not resolve correctly
        scripts.unshift("https://vlocity-aurang-poc-dev-ed.my.salesforce.com/resource/angularjs");

Is there a right way to get the resource to load correctly in the Lighting Out context?
Or does the library needs an update to handle this scenario correctly?

**Comment by dutchchasman** [comment]
This is indeed a bug and the root cause is as you expected (relative URL to /resource/angularjs needs to be absolutized to be requested on the Aura domain).

**Comment by AndreyVol** [comment]
By the way, the same problem applies to loading things like static resources.

**Comment by dutchchasman** [comment]
Yes. Understood - I'm looking at enhancing the Lightning framework to have it provide a way to handle this.

**Comment by Szandor72** [comment]
Same use case: Built an Heroku Node App for Lightning Out Demo. Static Resources aren't dealt with properly. No difference between $Ressource or explicit /path/
